packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -86,6 +86,7 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     double mainAxisMargin = 0.0,
     double crossAxisMargin = 0.0,
     Radius? radius,
+    Radius? trackRadius,
     OutlinedBorder? shape,
```

Comment:
I believe `trackShape` should also be added

Reply:
Done

Reply:
Can you explain the need for `trackShape` @bdlukaa? It is not mentioned in #97876
I am cautious about adding more and more parameters to this widget.

Reply:
Just in case someone wants a very custom scrollbar. For my use case, only `trackRadius` should all, but, since in Flutter we should have control over every pixel on the screen, I believe `trackShape` is a good parameter for this widget. 

We already have `shape` for the thumb, why not `trackShape`?

Reply:
Our engineering philosophy specifies

> Avoid implementing features you don’t need. 

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#lazy-programming

Asking `why not ...?` is a slippery slope to a large and complicated API. Imagine a Scrollbar with 100 different parameters... 😵‍💫

You are right, in Flutter you can control every pixel on the screen, that does not mean we need to do so in the framework. You can certainly extend the scrollbar class in your own code, or even publish a package. There are a number of scrollbar packages on pub.dev with a variety of special features.

The framework focuses on providing fundamental core UI features, but anyone can change and modify as they like in their own project. You are very right that you can control every pixel as you desire. :)

Reply:
i agree @Piinks' point. we may add it in the future if a user has a specific use case. i'll go ahead and remove this change.

Reply:
Tests are now failing @werainkhatri